echo/frontend/src/components/form/MarkdownWYSIWYG/styles.css

```
@@ -1511,23 +1432,23 @@
 
-._imageWrapper_uazmk_922 {
+._imageWrapper_sects_920 {
   display: inline-block;
   position: relative;
 }
 
-._imageWrapper_uazmk_922[draggable="true"] {
+._imageWrapper_sects_920[draggable='true'] {
   cursor: move;
   /* fallback if grab cursor is unsupported */
   cursor: grab;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

cursor is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)